server_config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SERVER_CONFIG_H
+#define SERVER_CONFIG_H
+
+#include "config_parser.h"
+#include <unordered_map>
+
+class ServerConfig {
+  public:
+  	ServerConfig(NginxConfig config);
+    ~ServerConfig();
+    std::string propertyLookUp(std::string propertyName);
+
+  private:
+  	void printPropertiesMap();
```

Comment:
Text editors are not always consistent; it was 2 spaces above and now suddenly 4 spaces. xD